UpdateCgnOrchestrator/index.ts

```diff
@@ -58,14 +55,14 @@ const trackExceptionAndThrow = (
   throw new Error(String(err));
 };
 
-const getMessageType = (cgnStatus: CgnStatus) => {
-  if (CgnRevokedStatus.is(cgnStatus)) {
-    return "CgnRevokedStatus";
+const getMessageType = (card: Card) => {
+  if (CardRevoked.is(card)) {
+    return "CardRevoked";
   }
-  if (CgnExpiredStatus.is(cgnStatus)) {
-    return "CgnExpiredStatus";
+  if (CardExpired.is(card)) {
+    return "CardExpired";
   } else {
-    return "CgnActivatedStatus";
+    return "CardActivated";
   }
 };
```

Comment:
This functions return `"CardActivated"` if card is `CardPending`. Maybe the function should throw an error
```typescript
const getMessageType = (card: CardRevoked | CardExpired | CardActivated) => {
  if (CardRevoked.is(card)) {
    return "CardRevoked";
  }
  if (CardExpired.is(card)) {
    return "CardExpired";
  }
  if (CardActivated.is(card)) {
    return "CardActivated";
  }
  throw new Error("Unexpected Card status");
};
```
or `assertNever`